### PR TITLE
fix create topic page number inputs validation

### PR DESCRIPTION
**Pull request by rAzizbekyan**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [x] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/common/Input/Input.tsx [hunk 1]
```diff
@@ -35,10 +36,21 @@ const Input: React.FC<InputProps> = ({
           inputSize={inputSize}
           {...methods.register(name, { ...hookFormOptions })}
           hasLeftIcon={!!leftIcon}
+          onKeyDown={(e) => {
+            if (e.key === 'e' && type === 'number') {
```
Comment (rAzizbekyan) [inline kafka-ui-react-app/src/components/common/Input/Input.tsx:40]: @workshur it's not letting to type letter 'e' but when copy pasting work containing letter 'e' it's not working, for that I can use e.keyCode or e.which but those two are deprecated, so should we keep this as it is or solve using deprecated methods?

Comment (workshur) [reply kafka-ui-react-app/src/components/common/Input/Input.tsx:40]: @rAzizbekyan  can we use pattern here? something like `pattern="^[-/d]/d*$"` for `integer only` input

Comment (rAzizbekyan) [reply kafka-ui-react-app/src/components/common/Input/Input.tsx:40]: it's not solving the problem as we need to not let letter 'e' to be typed,but via pattern we let 'e' to be typed but got respective error for that

**Comment by workshur** [comment]
will be fixed in https://github.com/provectus/kafka-ui/pull/2508

**Comment by workshur** [comment]
https://github.com/provectus/kafka-ui/pull/2508

**Pull request closed without merging**